samples/server/petstore/haskell-servant/lib/OpenAPIPetstore/Types.hs

```diff
@@ -6,8 +6,6 @@
 module OpenAPIPetstore.Types (
   ApiResponse (..),
   Category (..),
-  InlineObject (..),
```

Comment:
Why are these objects being removed?

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/4058/commits/9724c6d6e5dddc6ffe853cf1548fd484cde61f02

Those are the only changes I did to the samples. Those samples were not up to date so it seems that some time ago that line vanished. That change was made on updating the golden files to the latest master